assemblies/assembly-configuring-rhdh-events-module.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+
+[id="configuring-rhdh-events-module"]
+
+= Configuring {product} Events Module
+
+Without an Events system,
+changes you make to an external source through your Git provider are reflected in your {product-very-short} instance after a delay.
+You can remove the waiting time for entities ingestion by configuring the Events Module,
+and have the changes appear in your {product-very-short} instance immediately.
+This is a Technology Preview feature.
```

Comment:
I believe this is a Developer Preview feature as @schultzp2020 previously mentioned